controllers/kubevirtmachine_controller_test.go

```diff
@@ -298,12 +304,11 @@ var _ = Describe("reconcile a kubevirt machine", func() {
 
 		setupClient(machineFactoryMock, objects)
 
-		machineMock.EXPECT().Delete().Return(nil).Times(2)
 		machineMock.EXPECT().Exists().Return(true).Times(1)
-		machineMock.EXPECT().IsReady().Return(false).Times(2)
-		machineMock.EXPECT().Address().Return("1.1.1.1").Times(1)
-		machineMock.EXPECT().SupportsCheckingIsBootstrapped().Return(false).Times(1)
-		machineMock.EXPECT().GenerateProviderID().Return("abc", nil).Times(1)
+		machineMock.EXPECT().IsReady().Return(false).AnyTimes()
+		machineMock.EXPECT().Address().Return("1.1.1.1").AnyTimes()
+		machineMock.EXPECT().SupportsCheckingIsBootstrapped().Return(false).AnyTimes()
+		machineMock.EXPECT().GenerateProviderID().Return("abc", nil).AnyTimes()
```

Comment:
does `Times()` not work anymore with v2? If so, that's not great

Reply:
They do. They works too good. It now enfoced by ginkgo v2. I could changed it to any value I wanted in v1, and the test passed.

Reply:
lol... you're right. I can change the Times() to anything and it passes for me locally to with v1. ha. that's useless.